api/src/main/java/org/pmiops/workbench/utils/WorkspaceMapper.java

```diff
@@ -39,11 +43,24 @@
   @Mapping(target = "otherDisseminateResearchFindings", source = "disseminateResearchOther")
   ResearchPurpose workspaceToResearchPurpose(DbWorkspace dbWorkspace);
 
-  default Set<Short> map(List<SpecificPopulationEnum> value) {
-    return value.stream()
-        .map(DbStorageEnums::specificPopulationToStorage)
-        .collect(ImmutableSet.toImmutableSet());
-  }
+  @Mapping(target = "approved", ignore = true)
```

Comment:
Makes sense. Though I feel like presentation-layer considerations shouldn't really drive the object structure on the backend, since in principle we could have other clients that might be interested. 🤷‍♂

Reply:
sure, this tripped me up but I believe its only supposed to be set one time and never again. IDK why that is but all the code and tests I've read seem to point in that direction.

Reply:
Ideally, we should be letting the mapper run its course and handle business logic in the controller but that was looking like a much bigger refactoring effort.

Reply:
I've been meaning to start a pet peeves document on the Workspace and its handling up and down the stack. This will help with that if I ever get around to it.